uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/tables/PagedTable.java

```diff
@@ -28,9 +28,13 @@
 import org.uberfire.ext.services.shared.preferences.GridGlobalPreferences;
 import org.uberfire.ext.services.shared.preferences.GridPreferencesStore;
 import org.uberfire.ext.widgets.common.client.resources.i18n.CommonConstants;
+import org.uberfire.ext.widgets.table.client.PagedTableHelper;
+import org.uberfire.ext.widgets.table.client.UberfireSimplePager;
 
 /**
- * Widget that shows rows of paged data.
+ * Paged Table Widget that stores user preferences.
+ * If you doesn't need persist the preferences,
+ * take a look at UberfirePagedTable.
  */
 public class PagedTable<T>
```

Comment:
Paged Table == Uberfire Paged Table + Preferences Persistence.

In the future with the new Preferences Persistence  (part of qmx/porcelli work) we can hook that API to UberfireTable and probably remove this class.